client/lobby/CSelectionBase.cpp

```diff
@@ -93,13 +93,27 @@ CSelectionBase::CSelectionBase(ESelectionScreen type)
 	if(screenType == ESelectionScreen::campaignList)
 	{
 		setBackground(ImagePath::builtin("CamCust.bmp"));
+		pos = background->center();
 	}
 	else
 	{
-		const JsonVector & bgNames = CMainMenuConfig::get().getConfig()["game-select"].Vector();
+		const JsonNode& gameSelectConfig = CMainMenuConfig::get().getConfig()["scenario-selection"];
+		const JsonVector& bgNames = gameSelectConfig["background"].Vector();
```

Comment:
Please add fallback so game won't crash if player has 1.6 mod

Reply:
Done in latest commit